assets/hmr.js

```diff
@@ -7,6 +7,7 @@ function reload() {
 }
 
 const REGISTERED_MODULES = {};
+const socket = new WebSocket('ws://localhost:12321/');
```

Comment:
A quick question:
Lets say we are developing on a remote machine which has an ip like 192.168.1.1
And yarn start is runing
when we try to access from our own machine with http://192.168.1.1 isn' this connection fails?

Reply:
That does feel like a totally different issue since this would've failed before as well

Reply:
Yeah you are right, its not this pr's issue my bad 👍

Reply:
No worries, we should look at fixing that though, this could be done with a `replace` when we serve the `HMR` code if we see a custom IP in the config